.github/workflows/end-to-end-tests.yml

```
@@ -0,0 +1,36 @@
+name: "end-to-end_tests"
+on:
+    push:
+        branches: [develop]
+    pull_request:
+        branches: [develop]
+jobs:
+    "end-to-end_tests":
```

This could probably just be `end-to-end`?

```suggestion
    end-to-end:
```